AnkiDroid/src/main/java/com/ichi2/anki/CoroutineHelpers.kt

```
@@ -57,15 +57,17 @@ suspend fun <T> FragmentActivity.runCatchingTask(
     try {
         return block()
     } catch (exc: CancellationException) {
-        // do nothing
+        Timber.w(exc)
+        // Rethrowing cancellation exception to notify the parent
+        throw exc
```

This is a functional change and should be explained, ideally in a separate commit. 

All other paths ignore the exception

---

Okay will make a new PR, probably with some more enhancement.